app/models/devise_token_auth/concerns/user.rb

```diff
@@ -112,10 +112,6 @@ def valid_token?(token, client = 'default')
     false
   end
 
-  # this must be done from the controller so that additional params
-  # can be passed on from the client
-  def send_confirmation_notification?; false; end
-
```

Comment:
I think we don't need to override this code.
Because [original code](https://github.com/heartcombo/devise/blob/eed641d2bea11839ab13e943660da41cad14314d/lib/devise/models/confirmable.rb#L305-L307) depends on `@skip_confirmation_notification` and this gem [set it in a controller](https://github.com/lynndylanhurley/devise_token_auth/blob/5ef7197d5e2ca50f7fc92a7b04884341759b6e87/app/controllers/devise_token_auth/registrations_controller.rb#L38-L41).